client/lib/upgrades/actions/purchases.js

```diff
@@ -53,7 +59,9 @@ function cancelPrivateRegistration( purchaseId, onComplete ) {
 			Dispatcher.handleServerAction( {
 				type: ActionTypes.PRIVACY_PROTECTION_CANCEL_FAILED,
 				purchaseId,
-				error: error.message || i18n.translate( 'There was a problem canceling this private registration. Please try again later or contact support.' )
+				error: i18n.translate( 'There was a problem canceling this private registration. Please try again later or {{a}}contact support{{/a}}.', {
```

Comment:
What's the error message? If it's broken we should probably fix it.

I don't necessarily agree, unless the error message is about a technical error that users won't be able to understand. Otherwise I think we should display the message returned by the API, or update it if it doesn't do the job. Note a custom message is also useful for support when they are contacted by users and need to investigate a problem.